go.mod

```diff
@@ -5,7 +5,7 @@ go 1.19
 require (
 	cloud.google.com/go/container v1.14.0
 	cloud.google.com/go/firestore v1.9.0
-	cloud.google.com/go/iam v0.13.0
+	cloud.google.com/go/iam v1.0.0
```

Comment:
Just calling out this jump from 0.x to 1.0.

Reply:
https://github.com/googleapis/google-cloud-go/blob/main/iam/CHANGES.md
there are no changes between v0.13 and v1.0, just promoting to to v1 as I understand